.gitignore

```diff
@@ -92,7 +92,9 @@ target
 .ccls-cache/
 
 python/venv
+python/.venv
 test_data/venv
+test_data/.venv
```

Comment:
should we also just add `docs/.venv` instead of doing `uv pip install --system -r docs/requirements.txt`?